editor/js/editor-libs/telemetry.js

```diff
@@ -0,0 +1,68 @@
+import { postParentMessage } from "./events.js";
+
+const ACTION_COUNTS_KEY = "action-counts";
+
+function readActionsCounts() {
+  try {
+    const value = JSON.parse(localStorage.getItem(ACTION_COUNTS_KEY));
+    if (value && value.href === window.location.href && value.counts) {
+      return value.counts;
+    }
+  } catch (e) {
+    window.console.warn("Unable to read action counts from localStorage", e);
+  }
+
+  return {};
+}
+
+function persistActionCounts(counts) {
+  try {
+    localStorage.setItem(
+      ACTION_COUNTS_KEY,
+      JSON.stringify({
+        href: window.location.href,
+        counts,
+      })
+    );
+  } catch (e) {
+    window.console.warn("Unable to write action counts to localStorage", e);
```

Comment:
Could we make `window.console.warn` consistent with `console.warn` in `events.js`? `warn` function is not overwritten in `console.js`, so the output shouldn't be shown in the JS/WAT console.

Reply:
Resolved via ca746b645a0dcfc05d1c06f8b654d002751c8354 and c85f280bbed7194bd999224a9d8f8c8a1b4775fe.